### DISCUSSION TITLE
Real World Data Science - The many ‘dialects’ of data visualization: Alberto Cairo and ‘The Art of Insight’

### DISCUSSION THREAD

**Discussion by giscus** [realworlddatascience/realworlddatascience.github.io#50] [General]
# Real World Data Science - The many ‘dialects’ of data visualization: Alberto Cairo and ‘The Art of Insight’

In his upcoming new book, journalist and designer Alberto Cairo explores the idea of ‘data visualization as language’, one with many different dialects – statistical, journalistic, artistic, and so on. These dialects are not mutually exclusive, he says. So, what might they learn from each other?

https://realworlddatascience.net/viewpoints/interviews/posts/2023/08/01/alberto-cairo.html

**Comment by brtarran** [comment]
Following up on the Alberto Cairo interview, RWDS readers might also find this new RSS publication of interest: [Best Practices for Data Visualisation](https://royal-statistical-society.github.io/datavisguide/).
